sisyphus/localengine.py

```diff
@@ -109,14 +112,18 @@ def start_engine(self):
     def get_default_rqmt(self, task):
         return LOCAL_DEFAULTS
 
-    def start_task(self, task):
+    def start_task(self, task, selected_gpus):
         """
         :param TaskQueueInstance task:
+        :param str selected_gpus:
         :rtype: psutil.Process
         """
         # Start new task
         call = task.call[:]
-        sp = subprocess.Popen(call, start_new_session=True)
+        env = os.environ.copy()
+        if len(selected_gpus) > 0:
```

Comment:
more pythonic:
```
if selected_gpus:
```
Also: wouldn't it be better to always set it even if no gpu is selected? This way a job cannot without explicitly setting CUDA_VISIBLE_DEVICES itself take the GPU.

Reply:
I think the empty string is the normal way, or not?

Reply:
CUDA_VISIBLE_DEVICES is now set to empty string if no gpu is requested